Pages/VolunteerRegistration.razor.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using Karma.Models;
+using MatBlazor;
+
+namespace Karma.Pages
+{
+    public partial class VolunteerRegistration
+    {
+        private String Name { get; set; } = "";
```

Comment:
string

Reply:
Yeah, I also noticed the capital strings. Changed them.